indico/htdocs/js/indico/jquery/ajaxdialog.js

```diff
@@ -81,6 +82,10 @@
         loadDialog();
 
         function loadDialog() {
+            if (options.content) {
+                showDialog({js: '', html: options.content[0].outerHTML});
```

Comment:
What's wrong with passing the jquery object? Also, if it's content I would definitely not expect that the topmost element is the container of my content.